### ISSUE TITLE
Zeus does not work with lnbits lndhub extension

### ISSUE THREAD

**Issue by petrkr** [ZeusLN/zeus#919]
Zeus with LNBits LNDHub extension does not show transactions. On server side ti shows 
```
uvicorn[669815]: INFO:     - "GET /lndhub/ext/v1/transactions HTTP/1.1" 404 Not Found
```

Do not know if zeus uses old API, or lnbits does not implemented new API. Results is it does not work together...
BlueWallet works fine with same account

Since I do not know who messed it up, I placed issue also on lnbits side, so you can discuss who will repair it :) https://github.com/lnbits/lnbits-legend/issues/577

**Comment by petrkr** [comment]
Based on LNDHub documentation, it does not support API call v1/transactions

I think problem is because LNDHub backend extends class LND which contain that API  Call.. Even there is .catch if call was not handled well, it seems something messing up...

Do not know if there is another problem, typescript is not my strong programming language.

I guess getTransactions is method to get on-chain TXs, so maybe LNDHub backend can override this API call and return just empty list (I think LNDHub does not implement on-chain TX at all, so this call is useless)

**Comment by petrkr** [comment]
Now I discovered another behavior on new phone:

swipe up to see transactions:

there is animation seems like "loading"... and it waits like 30sec after those 30sec I can see GET on web server and right after it shows "no transactions" (in this case it is right, because it is new wallet... later I will try send some tx there...)

Server during that wait is NOT loaded more than 2% of CPU.

I will investigate more... Some debug logs would help :(

from ADB Logcat I have only some buffer logs called every 1 sec

```
05-07 16:03:57.394   509   597 I BufferQueueProducer: [app.zeusln.zeus/app.zeusln.zeus.MainActivity#0](this:0xb40000709af2af58,id:669,api:1,p:23630,c:509) queueBuffer: fps=33.42 dur=1017.27 max=34.71 min=18.49
05-07 16:03:58.423   509   598 I BufferQueueProducer: [app.zeusln.zeus/app.zeusln.zeus.MainActivity#0](this:0xb40000709af2af58,id:669,api:1,p:23630,c:509) queueBuffer: fps=33.02 dur=1029.53 max=35.31 min=16.52
05-07 16:03:59.436   509   597 I BufferQueueProducer: [app.zeusln.zeus/app.zeusln.zeus.MainActivity#0](this:0xb40000709af2af58,id:669,api:1,p:23630,c:509) queueBuffer: fps=34.57 dur=1012.58 max=34.84 min=16.60
```

**Comment by kaloudis** [comment]
@petrkr this should be fixed in v0.6.5. We cut rc1 last night.

**Comment by rasodu** [comment]
Is there an ETA on when the update will be release on Google Play store?
